components/layout/Meta.js

```diff
@@ -31,6 +34,12 @@ const Meta = ({ title, desc }) => (
       <script async src="/static/js/ga.js" />
```

Comment:
async isn't blocking? Since this is all just GA code, async would be better than defer, ya?